src/Symfony/Component/Validator/Mapping/Cache/DoctrineCache.php

```diff
@@ -14,10 +14,14 @@
 use Doctrine\Common\Cache\Cache;
 use Symfony\Component\Validator\Mapping\ClassMetadata;
 
+@trigger_error(sprintf('%s is deprecated in Symfony 4.4 and will be removed in 5.0', DoctrineCache::class), E_USER_DEPRECATED);
```

Comment:
`The "%s" class is deprecated since Symfony 4.4.`

Reply:
We are not supposed to put the "removed" part anymore as this is implicit, please check https://symfony.com/doc/current/contributing/code/conventions.html

Reply:
Oh, I didn't know that. I'll remove the part. Thanks.